service.go

```diff
@@ -196,7 +196,7 @@ func makeNewAddressable() (*models.Addressable, error) {
 			common.LoggingClient.Info(fmt.Sprintf("Addressable %s doesn't exist, creating a new one", common.ServiceName))
 			millis := time.Now().UnixNano() / int64(time.Millisecond)
 			addr = models.Addressable{
-				BaseObject: models.BaseObject{
+				Timestamps: models.Timestamps{
```

Comment:
Is use of Timestamps model intended to be included as part of this changeset?

Reply:
Yes.

Reply:
Thanks, I see it preceded these changes.
LGTM;

Reply:
FYI -- Found through testing that entries in `example/cmd/device-simple/res/Simple-Driver.yaml` also needed to be changed. Verified commands are now properly created in core-metadata when device service is started.